dashboards/sap-hana/metadata.yaml

```diff
@@ -8,4 +8,12 @@ sample_dashboards:
       There is also a card with links to docs and the SAP HANA logs in Cloud Logging.
     related_integrations:
       - id: saphana
-        platform: GCE
\ No newline at end of file
+        platform: GCE
+  -
+    category: SAP HANA
+    id: performance
+    display_name: SAP HANA Performance Monitoring
+    description: "This dashboard has 8 charts to display the metrics ingested via the HANA Monitoring Agent SAP HANA V2.0. The metrics include Instance Memory, Schema Records, Connections, and Rowstore Memory Size etc."
+    related_integrations:
+      - id: saphana
+        platform: GCE
```

Comment:
This is for dashboards with metrics collected by the Ops Agent, please remove.

Reply:
done